### PR TITLE
Added GitHub Actions

### PR DESCRIPTION
**Pull request by daminijain23**
Added -
release.yml
tox-test.yml

Deleted - 
.travis.yml

https://projects.engineering.redhat.com/browse/RHELDST-7266

### CODE REVIEW

File: .github/workflows/tox-test.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Tox tests
+
+on: [push, pull_request]
+
+jobs:
```
Comment (rajulkumar) [inline .github/workflows/tox-test.yml:5]: This should also include a Py27 job.

**Review by rajulkumar — CHANGES_REQUESTED** [review]
As suggested in other review, can the commits be squashed and a descriptive commit message be added.
Also, you may update the tox.ini and retire travis specific job in this or another commit.

**Comment by daminijain23** [comment]
@rajulkumar 
1. Added py27 job
2. As suggested in other repo review - removed codecov token  from tox-test.ymland change pypi_password to pypi token in release.yml.
3. squashed commit.
4. Removed travis related jobs in the same commit.

File: .github/workflows/tox-test.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e py27
+  py38:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install RPM
+        run: |
+          sudo apt-get install -y rpm
```
Comment (rohanpm) [inline .github/workflows/tox-test.yml:28]: Minor - these could all be a single command. It would be slightly more efficient.

**Review by rohanpm — APPROVED** [review]

**Review by rajulkumar — CHANGES_REQUESTED** [review]
[tox]
envlist = py27, py36

py36 should be changed to py38 as that's what is running in actions to avoid inconsistency. Else might introduce some confusion with tox running fine as it's using py36 but the CI running on py38 might have some errors.

**Review by rajulkumar — APPROVED** [review]

**Pull request merged**